src/turtle.rs

```diff
@@ -1156,4 +1158,26 @@ mod tests {
         assert_eq!(turtle.position()[1].round(), 100.0);
         assert_eq!(turtle.heading(), 51.0);
     }
+
+    #[test]
+    fn turn_towards() {
+        let mut turtle = Turtle::new();
+        turtle.set_speed("instant");
+        let mut last_delta_angle = 0.0;
```

Comment:
Please replace this line with the following code to get rid of the warning that occurs when this compiles.

```rust
        // This variable is used in the assert!(), but Rust still warns us because its value isn't
        // used to do anything other than that. We ignore this error since this is okay for tests.
        #[allow(unused_assignments)]
        let mut last_delta_angle = 0.0;
```